hooks/useEstimateGasCost.ts

```diff
@@ -41,39 +41,34 @@ export default function useEstimateGasCost() {
 	);
 
 	const estimateEthersContractTxCost = useCallback(
-		async (contract: Contract, method: string, params: any[]): Promise<Wei> => {
+		async (
+			contract: Contract,
+			method: string,
+			params: any[],
+			buffer?: number
```

Comment:
I like the idea you were discussing as buffer as a percent, makes it easier to apply to any tx then, what do you think?